### PR TITLE
Feature/popout location Map 70

### PR DESCRIPTION
**Pull request by dobbi-free**
Pictures are not loaded because of CORS

### CODE REVIEW

File: frontend/src/components/LocationPopOut/LocationPopOut.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    >
+      <StyledTooltip>
+        <StyledBox>
+          <StyledMediaBox>
+            {isLoading ? (
+              <CircularProgress />
+            ) : (
+              <CardMedia
+                sx={{ borderRadius: '20px' }}
+                src={locationData.photoSrc === '' ? img : locationData.photoSrc}
```
Comment (Yojuks) [inline frontend/src/components/LocationPopOut/LocationPopOut.tsx]: maybe - if photoSrc is undefined - then you should add alt - where you will add text with the name of image - like name of location

File: frontend/src/components/Map/index.tsx [hunk 2]
```diff
@@ -83,7 +84,11 @@ function Map() {
         style={{ height: '100vh' }}
       >
         <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
+
         <MyZoomComponent />
+        {locations.map(({ _id, coordinates }) => (
+          <LocationPopOut key={_id} id={_id} coordinates={coordinates} />
```
Comment (Yojuks) [inline frontend/src/components/Map/index.tsx:90]: why do you need attribute id={_id}?

**Review by ghost — APPROVED** [review]

**Review by steadyGuy — APPROVED** [review]
Looks good. But for svgs, I would rather create a separate component. With that, it will be much easier to customize it in the future.

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: frontend/src/components/Map/index.tsx (hunk 2 above)
Comment (dobbi-free) [reply frontend/src/components/Map/index.tsx:90]: I need to fetch data by id

File: frontend/src/components/LocationPopOut/LocationPopOut.tsx (hunk 1 above)
Comment (dobbi-free) [reply frontend/src/components/LocationPopOut/LocationPopOut.tsx]: done

**Review by Yojuks — APPROVED** [review]

**Pull request merged**